src/test/java/picard/nio/GATKBucketUtilsTest.java

```diff
@@ -0,0 +1,29 @@
+package picard.nio;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import org.testng.Assert;
+
+import java.net.URI;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+public class GATKBucketUtilsTest {
```

Comment:
Why move this tests here?

Reply:
Copied this here because I didn't see another test to catch if support for underscores in bucket names regressed.

I added the test after seeing "Added or modified tests to cover changes and any new functionality" in the checklist in the [issues template](https://github.com/broadinstitute/picard/blob/af057c4bab4a7af97506b64027c14f05de305b52/.github/PULL_REQUEST_TEMPLATE.md?plain=1#L16).